### DISCUSSION TITLE
Is there a way to statically generate css for text styles?

### DISCUSSION THREAD

**Discussion by Jonas-C** [chakra-ui/panda#2914] [Q&A] [answered]
Hi! Our project is using panda with minimal css props. However, we'd like to expose a couple of props on our `Text` component for convenience, namely a `textStyle` prop. However, this is not always picked up by the panda extractor, and we end up in situations where the component has the correct class name, but no corresponding css class. Is there a way to statically generate css for text styles?

**Comment by segunadebayo** [comment, marked as answer]
You can use the `staticCss` config property to pre-generate any style or recipe.

```tsx
import { defineConfig } from "@pandacss/dev";

export const config = defineConfig({
  // ...
  staticCss: {
    css: [{ properties: { textStyle: ['*'] } }],
  },
});
```
